WORKSPACE

```diff
@@ -105,6 +105,28 @@ load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 bazel_skylib_workspace()
 ###############################################################################
 
+
+# Group the sources of the library so that CMake rule have access to it
+all_content = """filegroup(name = "all", srcs = glob(["**"]), visibility = ["//visibility:public"])"""
+
+# Rule repository
+http_archive(
+   name = "rules_foreign_cc",
+   strip_prefix = "rules_foreign_cc-master",
+   url = "https://github.com/bazelbuild/rules_foreign_cc/archive/master.zip",
+)
+
+load("@rules_foreign_cc//:workspace_definitions.bzl", "rules_foreign_cc_dependencies")
+
+rules_foreign_cc_dependencies()
+
+# spirv source code repository
+new_local_repository(
+   name = "spirv_cross",
+   path = "third_party/spirv_cross",
+   build_file_content = all_content,
+)
```

Comment:
As I mention in another comment, if we add a `BUILD` file instead of using `cmake_external`, we can drop these `WORKSPACE` changes.